packages/react-data-grid/src/Grid.js

```diff
@@ -86,6 +80,50 @@ const Grid = createReactClass({
     };
   },
 
+  _onScroll() {
```

Comment:
`GridScrollMixin` is only used by the `Grid` component so moved the logic into the `Grid` component

`DOMMetrics.MetricsComputatorMixin` defines the `childContextTypes` to implicitly pass `metricsComputator` to all the descendant components. `ReactDataGrid` component is using the same mixin so `Grid` component does not need to add it. It can be removed safely

https://reactjs.org/docs/context.html